<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportSetupDialog</class>
 <widget class="QDialog" name="ReportSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>668</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="4">
    <widget class="QLabel" name="lblEndPayDate">
     <property name="text">
      <string>по</string>
     </property>
     <property name="buddy">
      <cstring>edtEndPayDate</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="9">
    <widget class="QCheckBox" name="chkOnlyPermanentAttach">
     <property name="text">
      <string>&amp;Прикреплённые к базовому ЛПУ</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="CDateEdit" name="edtBegPayDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="27" column="1" colspan="2">
    <widget class="QComboBox" name="cmbMKBFilter">
     <item>
      <property name="text">
       <string>Игнор.</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Интервал</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="24" column="8" colspan="2">
    <spacer name="spcActionType">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>81</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1" colspan="9">
    <widget class="QCheckBox" name="chkOnlyPayedEvents">
     <property name="text">
      <string>Только опла&amp;ченные</string>
     </property>
    </widget>
   </item>
   <item row="21" column="1" colspan="5">
    <widget class="QFrame" name="frmAge">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="edtAgeFrom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>по</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edtAgeTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeYears">
        <property name="text">
         <string>лет</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="lblWorkOrganisation">
     <property name="text">
      <string>Место работы</string>
     </property>
    </widget>
   </item>
   <item row="24" column="1" colspan="7">
    <widget class="CActionTypeComboBox" name="cmbActionType"/>
   </item>
   <item row="21" column="0">
    <widget class="QLabel" name="lblAge">
     <property name="text">
      <string>Возраст с</string>
     </property>
     <property name="buddy">
      <cstring>edtAgeFrom</cstring>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="9">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="9" column="6">
    <widget class="CDateEdit" name="edtEndPayDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lblInsurer">
     <property name="text">
      <string>СМО</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="23" column="0">
    <widget class="QLabel" name="lblActionTypeClass">
     <property name="text">
      <string>Кла&amp;сс</string>
     </property>
     <property name="buddy">
      <cstring>cmbActionTypeClass</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="9">
    <widget class="CRBComboBox" name="cmbStage"/>
   </item>
   <item row="8" column="1" colspan="9">
    <widget class="QCheckBox" name="chkDiagnosisType">
     <property name="text">
      <string>Учитывать сопуствующие диагнозы</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="lblSpeciality">
     <property name="text">
      <string>Специальность</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="9">
    <widget class="CInsurerComboBox" name="cmbInsurer"/>
   </item>
   <item row="20" column="2" colspan="8">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>141</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="lblWorkType">
     <property name="text">
      <string>По ОКВЭД</string>
     </property>
     <property name="buddy">
      <cstring>cmbWorkType</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="6">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblBegPayDate">
     <property name="text">
      <string>Оплачено в период с</string>
     </property>
     <property name="buddy">
      <cstring>edtBegPayDate</cstring>
     </property>
    </widget>
   </item>
   <item row="31" column="0" colspan="10">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="24" column="0">
    <widget class="QLabel" name="lblActionType">
     <property name="text">
      <string>&amp;Мероприятие</string>
     </property>
     <property name="buddy">
      <cstring>cmbActionType</cstring>
     </property>
    </widget>
   </item>
   <item row="30" column="0" colspan="10">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>428</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="22" column="0" colspan="10">
    <widget class="QCheckBox" name="chkActionClass">
     <property name="text">
      <string>Не учитывать класс</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="21" column="6" colspan="4">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="19" column="9">
    <widget class="QToolButton" name="btnSelectWorkOrganisation">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="27" column="5" colspan="2">
    <widget class="CICDCodeEdit" name="edtMKBTo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string>a00.00; </string>
     </property>
     <property name="text">
      <string>Z99.9</string>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="25" column="0">
    <widget class="QLabel" name="lblFinance">
     <property name="text">
      <string>Тип финанисирования</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1" colspan="9">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="25" column="1" colspan="7">
    <widget class="CRBComboBox" name="cmbFinance"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblStage">
     <property name="text">
      <string>Стадия</string>
     </property>
     <property name="buddy">
      <cstring>cmbStage</cstring>
     </property>
    </widget>
   </item>
   <item row="20" column="0">
    <widget class="QLabel" name="lblSex">
     <property name="text">
      <string>По&amp;л</string>
     </property>
     <property name="buddy">
      <cstring>cmbSex</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblEventType">
     <property name="text">
      <string>&amp;Тип обращения</string>
     </property>
     <property name="buddy">
      <cstring>cmbEventType</cstring>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>Врач</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="lblOwnership">
     <property name="text">
      <string>Собственность</string>
     </property>
     <property name="buddy">
      <cstring>cmbOwnership</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="6">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="27" column="0">
    <widget class="QLabel" name="lblMKBFilter">
     <property name="text">
      <string>Коды диагнозов по &amp;МКБ</string>
     </property>
     <property name="buddy">
      <cstring>cmbMKBFilter</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="9">
    <widget class="CRBComboBox" name="cmbEventType"/>
   </item>
   <item row="27" column="3" colspan="2">
    <widget class="CICDCodeEdit" name="edtMKBFrom">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMask">
      <string>a00.00; </string>
     </property>
     <property name="text">
      <string>A.</string>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="17" column="1" colspan="9">
    <widget class="QComboBox" name="cmbWorkType"/>
   </item>
   <item row="15" column="0" colspan="10">
    <widget class="QCheckBox" name="chkDetailPerson">
     <property name="text">
      <string>Детализировать по врачам</string>
     </property>
    </widget>
   </item>
   <item row="27" column="7" colspan="3">
    <spacer name="spacerMKBFilter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>117</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="26" column="0">
    <widget class="QLabel" name="lblContract">
     <property name="text">
      <string>Договор</string>
     </property>
    </widget>
   </item>
   <item row="26" column="1" colspan="7">
    <widget class="CContractComboBox" name="cmbContract"/>
   </item>
   <item row="23" column="1" colspan="7">
    <widget class="QComboBox" name="cmbActionTypeClass">
     <item>
      <property name="text">
       <string>Статус</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Диагностика</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Лечение</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Прочие мероприятия</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="20" column="1">
    <widget class="QComboBox" name="cmbSex">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ж</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="18" column="1" colspan="9">
    <widget class="QComboBox" name="cmbOwnership">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Бюджетная</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Частная</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Смешенная</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="23" column="8" colspan="2">
    <spacer name="spcActionTypeClass">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>31</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="1" colspan="9">
    <widget class="CRBComboBox" name="cmbSpeciality"/>
   </item>
   <item row="19" column="1" colspan="8">
    <widget class="COrgComboBox" name="cmbWorkOrganisation"/>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="chkOnlyNotPayedEvents">
     <property name="text">
      <string>Не оплаченные услуги</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chkOnlyEmployee">
     <property name="text">
      <string>Сотрудник</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="chkGroupByClients">
     <property name="text">
      <string>Группировать по пациентам</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="9">
    <widget class="CRBComboBox" name="cmbEventPurpose"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblEventPurpose">
     <property name="text">
      <string>Назначение обращения</string>
     </property>
     <property name="buddy">
      <cstring>cmbEventPurpose</cstring>
     </property>
    </widget>
   </item>
   <item row="28" column="9">
    <widget class="QToolButton" name="btnBrowseSourceDataFile">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="28" column="1" colspan="8">
    <widget class="QLineEdit" name="edtSourceDataFileName">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="28" column="0">
    <widget class="QCheckBox" name="chkIsDataFromFile">
     <property name="text">
      <string>Исходные данные из файла</string>
     </property>
    </widget>
   </item>
   <item row="29" column="0">
    <widget class="QLabel" name="lblOrgStrucutreAttachType">
     <property name="text">
      <string>Подразделение прикрепления</string>
     </property>
    </widget>
   </item>
   <item row="29" column="1" colspan="7">
    <widget class="COrgStructureComboBox" name="cmbOrgStructureAttachType"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes</header>
  </customwidget>
  <customwidget>
   <class>COrgComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CICDCodeEdit</class>
   <extends>QLineEdit</extends>
   <header>library/ICDCodeEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CInsurerComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CContractComboBox</class>
   <extends>QComboBox</extends>
   <header>Accounting/ContractComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CActionTypeComboBox</class>
   <extends>QComboBox</extends>
   <header>Events/ActionTypeComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbEventType</tabstop>
  <tabstop>cmbStage</tabstop>
  <tabstop>chkOnlyPermanentAttach</tabstop>
  <tabstop>chkOnlyPayedEvents</tabstop>
  <tabstop>edtBegPayDate</tabstop>
  <tabstop>cmbInsurer</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbSpeciality</tabstop>
  <tabstop>chkDetailPerson</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>cmbWorkType</tabstop>
  <tabstop>cmbOwnership</tabstop>
  <tabstop>cmbWorkOrganisation</tabstop>
  <tabstop>btnSelectWorkOrganisation</tabstop>
  <tabstop>cmbSex</tabstop>
  <tabstop>edtAgeFrom</tabstop>
  <tabstop>edtAgeTo</tabstop>
  <tabstop>chkActionClass</tabstop>
  <tabstop>cmbActionTypeClass</tabstop>
  <tabstop>cmbActionType</tabstop>
  <tabstop>cmbFinance</tabstop>
  <tabstop>cmbMKBFilter</tabstop>
  <tabstop>edtMKBFrom</tabstop>
  <tabstop>edtMKBTo</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>743</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>743</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOnlyPayedEvents</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtBegPayDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOnlyPayedEvents</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtEndPayDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkDetailPerson</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbPerson</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkActionClass</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbActionTypeClass</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>594</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkActionClass</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbActionType</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>625</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
